frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -63,17 +68,26 @@ const EditRecipeForm = ({ history }) => {
   const handleSubmit = () => {
     const parsedDuration = parseInt(duration, 10);
     const parsedServings = parseInt(servings, 10);
-
-    createRecipe({
-      title,
-      description,
-      difficulty,
-      duration: parsedDuration,
-      servings: parsedServings,
-      tags: selectedTagsWithId,
-      directions,
-      ingredients,
-    }).then(response => history.push(`/recipe/${response.data.id}`));
+    if (!title) {
+      setTitleError('Title cannot be blank');
+    } if (!description) {
+      setDescriptionError('Description cannot be blank');
+    } if (ingredients.length <= 1) {
+      setIngredientError('ingredients cannot be blank');
+    } if (directions.length <= 1) {
+      setDirectionsError('directions cannot be blank');
+    } else {
+      createRecipe({
```

Comment:
I think we might need something more here. As is, it looks like the only thing stopping `createRecipe` from getting called is `directions.length` being greater than one.